src/Adapters/Files/FilesAdapter.js

```diff
@@ -31,10 +31,18 @@ export class FilesAdapter {
    * @param {*} data - the buffer of data from the file
    * @param {string} contentType - the supposed contentType
    * @discussion the contentType can be undefined if the controller was not able to determine it
+   * @param {object} options - (Optional) options to be passed to file adapter (S3 File Adapter Only)
```

Comment:
nit: change the message to indicate that this may not be supported by all adapters and to check adapter documentation for support